lib/game_statistics.rb

```diff
@@ -23,4 +23,17 @@ def highest_total_score
     total_goals.max
   end
 
+  def percentage_home_wins
+    index = 0
+    home_wins = 0
+    data_size.times do
+      if @games_hash["home_goals"][index] > @games_hash["away_goals"][index]
+        home_wins += 1
+        index += 1
```

Comment:
I see an opportunity for refactoring.
**Possible Solution**
- if can be turned into loop where home_wins is the behavior inside.
- index += 1 outside of boolean loop but inside .times loop.

Reply:
Other solution (peer talked)
- delete line 32-34
-create index changer after if conditional

Reply:
Tried solution 2. Looks like that will work! Thank you so much for your suggestion!

I will consider additional ways to refactor this method!